soccer/gameplay/skills/goalside_mark.py

```diff
@@ -69,18 +62,14 @@ def execute_running(self):
         #   - 0 being close to ball, 1 close to mark pt
         self._target_point = None
         if mark_line_dist > self.mark_line_thresh:
-            self._target_point = mark_line.nearest_point(pos)
+            self._target_point = mark_line.nearest_point(self.robot.pos)
```

Comment:
self.robot can be none at times if we don't have enough robots to fill roles

Reply:
added self.robot is not None to the conditions required to execute_running